tests/unit/test_helpers.py

```diff
@@ -316,6 +319,80 @@ def test_json_object(self):
         # A JSON value, which is not an object
         self.assertFalse(is_json_object(json.loads('null')))
 
+    def test_ab_active_true(self):
+        """Check if we return the right view based on AB params."""
+        cookie = 'foo=bar; Path=/'
```

Comment:
Could we avoid the foo, foobar, bar everywhere? It makes the tests harder to read. Replace them with a value which looks more real(?).